### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by aepanchi**
### Description 
Replace myself with @bdmoore1 for doc-related PRs.

Fixes # - _issue number(s) if exists_

- [ ] I have reviewed my changes thoroughly before submitting this pull request.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation to reflect the changes, if necessary (updated in # - _add PR number_)
- [ ] The unit tests pass successfully.
- [ ] I have run it locally and tested the changes extensively.
- [ ] I have resolved any merge conflicts that might occur with the base branch.
- [ ] Git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/intel/scikit-learn-intelex/blob/main/CONTRIBUTING.md#pull-requests) for details)_
- [ ] I have added a respective label(s) to PR if I have a permission for that.

### CODE REVIEW

**Review by samir-nasibli — APPROVED** [review]

**Review by ethanglaser — CHANGES_REQUESTED** [review]
Same issue with sklearnex PR - does @bdmoore1 have write access to the repos?

**Comment by samir-nasibli** [comment]
> Same issue with sklearnex PR - does @bdmoore1 have write access to the repos?

Should this be a blocker?

**Review by Alexsandruss — APPROVED** [review]
